example/terraform/main.tf

```diff
@@ -5,12 +5,26 @@ provider "layer0" {
 }
 
 module "consul" {
-  source         = "github.com/quintilesims/consul/terraform"
-  environment_id = "${layer0_environment.dev.id}"
+  source         = "../../terraform"
+  environment_id = "${layer0_environment.consul.id}"
+}
+
+resource "layer0_environment" "consul" {
+  name = "consul"
+  // TODO: Remove hardcoded AMI
+  ami = "ami-f5fc2c8d"
+  min_count = 3
```

Comment:
in the future, would be nice to have `min_count` determined internally by scheduler...

i consider it dangerous that users need to manually set this based on sum of discrete ECS instances that will end up getting spun up.

Reply:
I agree this is not ideal, and we're talking about ways to make it less dangerous within our current Layer0 refactor. That being said, there is an argument for the explicitness here and how such apps are not designed to run on Layer0 in the first place.